instrumentation/net_http/lib/opentelemetry/instrumentation/net/http/patches/instrumentation.rb

```diff
@@ -11,6 +11,8 @@ module HTTP
         module Patches
           # Module to prepend to Net::HTTP for instrumentation
           module Instrumentation
+            include OpenTelemetry::SemanticConventions::Trace
```

Comment:
One think I worry about with mixing in the constants is the potential for conflicting with ones that _may_ exist in libraries. 

Perhaps mixing in only `OpenTelemetry::SemanticConventions` so that the constants are scoped to `Trace` or `Resource may be good enough?

Not sure how the @open-telemetry/ruby-approvers and @open-telemetry/ruby-maintainers feel about that.

Reply:
make sense. I can replace with a full path to constants.

Example:
`HTTP_STATUS` -> `OpenTelemetry::SemanticConventions::Trace::HTTP_STATUS`

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-ruby/pull/1083/commits/c697489673baa5982e2232905eabc5fb54a64674